data/preset/A.yaml

```diff
@@ -1,18 +1,53 @@
 ---
-defines:
-- &card
-  Card.Default
+vars:
+  app-banner:
+    shortdef: 'Banner.Dynamic(mode: full, halign: center)'
+
+  home-sets-arrangement:
+    shortdef: 'Arrangement.Overflow(orientation: horizontal)'
+  home-sets-card:
+    type: &card Card.Default
+    properties:
+      excluded_types:
+        - 0
+        - 2
+  home-allsets-arrangement:
+    type: Arrangement.SquareGuys
+    properties:
+      halign: center
+      valign: start
+      expand: true
+  home-allsets-card:
+    type: *card
+    properties:
+      excluded_types:
+        - 0
+        - 2
 
-- &main-arrangement-slot
-  arrangement:
+  set-articles-arrangement:
     type: Arrangement.SquareGuys
     properties:
       halign: center
       valign: start
       expand: true
-    slots:
-      card: *card
+  set-articles-card:
+    type: *card
```

Comment:
It seems to me that customizing this and `search-articles-card` separately would lead to apps with visual inconsistency. Same remark for `set-articles-arrangement` and `search-articles-arrangement`. (In general I'm in favour of using the API to enforce API users to do the right thing, unless there's a good reason why these need to be separated.)

Reply:
Let me double check if I understand properly, because I don't think I agree:

> It seems to me that customizing this and search-articles-card separately would lead to apps with visual inconsistency. Same remark for set-articles-arrangement and search-articles-arrangement.

If an app developer/designer decided that for its content it would be more appropriate to use Arrangement.List/Card.List for search results here (like we do our self in other presets), I don't see why we should prevent that possibility.

> (In general I'm in favour of using the API to enforce API users to do the right thing, unless there's a good reason why these need to be separated.)

As I see it, this feature is about providing more flexibility and visual diversity to our current and (more importantly) future presets. I don't see how preventing them for doing something like I mentioned above would be the right thing to do.

This pattern allows them to do that, and still allows them to use the same card arrangement everywhere if that is what they design.

Both @katelk28 and I reached this same pattern, so I think we need her thoughts on this as well.

(IMPORTANT) On the technical side, just to double check this is clear for others (if reading this), the example you provided:
```
vars:
  something-something-card: &main-card
    shortdef: 'Card.Default(...)'
  something-else-card: *main-card
```

This does not mean that `something-else-card` will get overridden when `something-something-card`. The `main-card` YAML anchor is simply there to save copy pasting `Card.Default` definitions all over the preset, but in the end, each of these variables are completely independent (just happen to have the same default value).